ONSdigital/eq-survey-runner
# eq-5 error messages template

![screen shot 2016-03-23 at 07 50 43](https://cloud.githubusercontent.com/assets/930398/13978844/0615a3ce-f0cc-11e5-9b55-67cd013c0e15.png)
### Changes
- Error messages displayed at top of page inside `.alert` with links to their respective response.
### Test
- `npm run compile`
- Submit form with errors
- Check that output is as per screenshot
